server/src/main/kotlin/com/tiden/flagship/payment/models/PaymentRequest.kt

```diff
@@ -2,12 +2,9 @@ package com.tiden.flagship.payment.models
 
 import kotlinx.serialization.Serializable
 
-// In-memory storage for Payments (temporary). Will eventually be persisted in DB
-val paymentStorage = mutableListOf<Payment>()
-
 // This annotation from kotlinx.serialization can work with Ktor to generate JSON representation for API responses automatically
 @Serializable
-data class Payment(
+data class PaymentRequest(
```

Comment:
non-blocking: as a follow-up, we may need to take in the merchant id from the frontend (possibly based on which  merchant's page the payment is being made from)